src/geometry/ArborX_DetailsAlgorithms.hpp

```diff
@@ -521,15 +521,24 @@ template <typename Point, typename Box>
 struct intersects<PointTag, BoxTag, Point, Box>
 {
   KOKKOS_FUNCTION static constexpr bool apply(Point const &point,
-                                              Box const &other)
+                                              Box const &box)
   {
     constexpr int DIM = GeometryTraits::dimension_v<Point>;
     for (int d = 0; d < DIM; ++d)
-      if (point[d] > other.maxCorner()[d] || point[d] < other.minCorner()[d])
+      if (point[d] > box.maxCorner()[d] || point[d] < box.minCorner()[d])
         return false;
     return true;
   }
 };
+template <typename Box, typename Point>
+struct intersects<BoxTag, PointTag, Box, Point>
+{
+  KOKKOS_FUNCTION static constexpr bool apply(Box const &box,
+                                              Point const &point)
+  {
+    return Details::intersects(point, box);
+  }
+};
```

Comment:
Had to add that because of some example that switch from a hierarchy based on boxes to hierarchy based on points.